### PR TITLE
Remove computeLedgerHash and isValidAddress from API

### PR DESCRIPTION
**Pull request by clark800**
isValidAddress will be provided by ripple-address-codec and computeLedgerHash will be provided by ripple-lib-serialization

### CODE REVIEW

**Comment by lumberj** [comment]
This doesn't actually remove the code for isValidAddress?

File: src/api/index.js [hunk 1]
```diff
@@ -31,7 +31,6 @@ const convertExceptions = require('./common').convertExceptions;
 const generateAddress = convertExceptions(common.generateAddress);
 const computeLedgerHash = require('./offline/ledgerhash');
 const getLedger = require('./ledger/ledger');
-const isValidAddress = common.isValidAddress;
```
Comment (lumberj) [inline src/api/index.js]: Is this being used anywhere in ripple-lib? If not, remove this from common?

**Comment by clark800** [comment]
Yeah, it's used internally

File: src/api/index.js (hunk 1 above)
Comment (clark800) [inline src/api/index.js]: Could remove it from common now, i'll do that

**Comment by lumberj** [comment]
Is the replacement for this already available?

**Comment by clark800** [comment]
There's a pull request for it: https://github.com/ripple/ripple-address-codec/pulls

**Comment by lumberj** [comment]
LGTM

**Pull request merged**